rmw_zenoh_cpp/package.xml

```diff
@@ -30,6 +30,8 @@
 
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
+  <test_depend>rclcpp</test_depend>
```

Comment:
not sure if we can do this here. This might generate a circular depencency. `rclcpp` it build based on the `rmw`.

we can create a `test_rmw_zenoh_cpp` new package and include the tests there